src/50_Samples_%26_Templates/Housing.html

```diff
@@ -0,0 +1,337 @@
+        font-weight: 400;
+        font-size: 24px;
+        vertical-align: middle;
+        float: left;
+    }
+    .related:hover {
+        background-color: #ccc;
+    }
+    .call-to-action {
+         margin: 0 16px;
+    }
+    @media (max-width: 320px) {
+        .call-to-action {
+            clear: both;
+            float: none;
+        }
+    }
+    .agency-button {
+        background-color: rgb(66,133,244);
+    }
+    .price-description {
+        color: green;
+        font-weight: 400;
+    }
+    #housing-header {
+        margin-bottom: 24px;
+        margin-top: 24px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+    }
+    .price-other {
+        font-weight: bold;
+    }
+    .contact {
+        display: flex;
+        margin-bottom: 16px;
+        margin-top: 16px;
+        align-items: center;
+        padding: 0 16px;
+    }
+    .contact h3{
+        margin-top: 0px;
+        margin-bottom: 0px;
+        padding: 0px;
+    }
+    .social-share {
+        line-height:36px;
+        display: flex;
+        padding-bottom: 16px;
+        padding-top: 16px;
+    }
+    amp-social-share {
+        margin-right: 8px;
+    }
+    .list-overflow {
+        position: absolute;
+        bottom: 0;
+    }
+    .related {
+        background-color: #f5f5f5;
+        display: block;
+        color: #111;
+        height: 75px;
+        padding: 0;
+        line-height: 75px;
+    }
+    .form-field-right {
+        border: 1px solid #dfdfdf;
+        position: relative;
+        background: white;
+        width: 200px;
+        width: 179px;
+    }
+    .form-field-right input {
+        border: none;
+    }
+    .form-field-left {
+        width: 200px;
+    }
+    .form-field {
+        display: flex;
+        margin-bottom: 16px;
+    }
+    </style>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    </head>
+    <body>
+        <div id="housing-header">
+            <h2>One bedroom flat for sale</h2>
+            <h4 class="price-description">£430,000</h4> 
+        </div>
+        <p id="housing-description"> Flowers court, 57 James Bond avenue, SW35YJ</p>
```

Comment:
I don't think this is the correct address. It should be the same as on the maps embed.